apps/src/p5lab/P5Wrapper.js

```diff
@@ -205,6 +205,7 @@ P5Wrapper.prototype.startExecution = function () {
   new window.p5(
     function (p5obj) {
       this.p5 = p5obj;
+      this.p5._onblur(); // This is to explicitly wipe out the downKeys object over in p5.js so no old keys hang around.
```

Comment:
nit: this comment should go above the function call, to match the styling in the rest of the file (and is our general practice, too).